<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>blackj</class>
 <widget class="QDialog" name="blackj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">justify-content: space-between;</string>
  </property>
  <widget class="QPushButton" name="blackjBackButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
  <widget class="QLabel" name="blackjDealerContent">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid red
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="blackjPlayerContent">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="blackjResultBar">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="blackjPlayButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">display: inline-block;</string>
   </property>
   <property name="text">
    <string>play</string>
   </property>
   <property name="autoRepeatDelay">
    <number>302</number>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>246</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="blackjBalanceTitle">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Balance:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="blackjBalanceNum">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="blackjDepositButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>deposit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>208</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="blackjHitButton">
      <property name="text">
       <string>Hit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="blackjStayButton">
      <property name="text">
       <string>Stay</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="blackjDoubleButton">
      <property name="text">
       <string>Double</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>268</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="blackjPrizeTitle_2">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Bet:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="blackjBetNum">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="blackjPrizeTitle">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Prize:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="blackjPrizeNum">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
